<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputChoiceWindow</class>
 <widget class="QWidget" name="OutputChoiceWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="back">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>370</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QPushButton" name="next">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>370</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
       <widget class="QLabel" name="title">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>601</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Output file selection</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="description">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>581</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This window allows you to select where you would like to save the output of the cut video.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
       <widget class="QLabel" name="error">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>581</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;The output file is invalid. Please select another one.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="tooltip">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>508</x>
          <y>289</y>
          <width>81</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;You have errors. Please correct them.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="inputFileText">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>230</y>
          <width>451</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Choose an input file...</string>
        </property>
       </widget>
       <widget class="QPushButton" name="inputFileButton">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>230</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Choose...</string>
        </property>
       </widget>
       <widget class="QLabel" name="properties">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>581</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Properties</string>
        </property>
       </widget>
       <widget class="QLabel" name="inputFileLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Input file:</string>
        </property>
       </widget>
       <widget class="QLabel" name="startTimeLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Start time:</string>
        </property>
       </widget>
       <widget class="QLabel" name="endTimeLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>End time:</string>
        </property>
       </widget>
       <widget class="QLabel" name="inputFileValue">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>100</y>
          <width>471</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>[The input file]</string>
        </property>
       </widget>
       <widget class="QLabel" name="startTimeValue">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>120</y>
          <width>471</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>[The start time to begin the cut]</string>
        </property>
       </widget>
       <widget class="QLabel" name="endTimeValue">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>140</y>
          <width>471</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>[The end time to end the cut]</string>
        </property>
       </widget>
       <widget class="QLabel" name="totalTimeLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Total time:</string>
        </property>
       </widget>
       <widget class="QLabel" name="totalTimeValue">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>160</y>
          <width>471</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>[The total time of the cut]</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
